cli/azd/internal/scaffold/scaffold.go

```diff
@@ -149,6 +149,13 @@ func ExecInfraFs(
 		return nil, fmt.Errorf("scaffolding main.parameters.json: %w", err)
 	}
 
+	if spec.AiFoundryProject != nil {
```

Comment:
Add a test to `scaffold_test.go` to make sure we're generating validate Bicep and it compiles cleanly.

Reply:
Added a little but noticed that somehow lint issues/warning are ignored in CI...  (don't know why).

When running this test locally from main branch I can see lint errors, but CI is not reporting anything.